MantidQt/CustomInterfaces/src/Indirect/AbsorptionCorrections.cpp

```diff
@@ -114,21 +117,15 @@ void AbsorptionCorrections::run() {
 
   // Set the correction workspace to keep the factors if desired
   bool keepCorrectionFactors = m_uiForm.ckKeepFactors->isChecked();
-  QString outputFactorsWsName = outputBaseName + "_" + sampleShape + "_Factors";
+  QString m_outputFactorsWsName =
```

Comment:
Do we want to define a new `m_outputFactorWsName` local variable here or should we just use the member variable already defined in the `.h`?